tests/test.sh

```diff
@@ -1476,11 +1476,12 @@ fi
 
 if test -z "$chroot_disabled"; then
     if test -z "$using_gcc"; then
+        # clang jobs don't need the gcc stderr workaround, just spit out the errors.
         run_ice "" "remote" 1 $TESTCXX -c syntaxerror.cpp
-        check_section_log_error icecc "local build forced by remote exception: Error 102 - command needs stdout/stderr workaround, recompiling locally"
```

Comment:
This removal is incorrect. It checks that the log message about stderr workaround is NOT present and errors out if it is.

Reply:
Ahh, the difference between the "error" and "message" functions is quite subtle.  Added back.